### PR TITLE
Add release note automation

### PR DESCRIPTION
**Pull request by abejgonzalez**
Follows the same type of automation done by https://github.com/chipsalliance/chisel/pull/3170. Note: This doesn't append to the `CHANGELOG.md` file.

**Related PRs / Issues**:

**Type of change**:
- [ ] Bug fix
- [ ] New feature
- [x] Other enhancement

**Impact**:
- [ ] RTL change
- [ ] Software change (RISC-V software)
- [ ] Build system change
- [x] Other

**Contributor Checklist**:
- [ ] Did you set `main` as the base branch?
- [ ] Is this PR's title suitable for inclusion in the changelog and have you added a `changelog:<topic>` label?
- [ ] Did you state the type-of-change/impact?
- [ ] Did you delete any extraneous prints/debugging code?
- [ ] Did you mark the PR with a `changelog:` label?
- [ ] (If applicable) Did you update the conda `.conda-lock.yml` file if you updated the conda requirements file?
- [ ] (If applicable) Did you add documentation for the feature?
- [ ] (If applicable) Did you add a test demonstrating the PR?

- [ ] (If applicable) Did you mark the PR as `Please Backport`?

### CODE REVIEW

**Review by jerryz123 — APPROVED** [review]
I don't really know what this is doing from this PR, but it seems fine at a glance.

**Comment by jerryz123** [comment]
Perhaps wait until post-release to merge this?

**Comment by abejgonzalez** [comment]
I'm interested to see if it works for this release (do we not have a changelog made already)?

**Pull request merged**
